src/parser/ad_parser.js

```diff
@@ -39,17 +49,24 @@ export function parseAd(adElement, emit) {
  * Parses an Inline
  * @param  {Object} ad Element - The VAST Inline element to parse.
  * @param  {Function} emit - Emit function used to trigger Warning event.
+ * @param  {Object} options - An optional Object of parameters to be used in the parsing process.
  * @emits  VASTParser#VAST-warning
  * @return {Object} ad - The ad object.
  */
-function parseInLine(adElement, emit) {
+function parseInLine(adElement, emit, { allowMultipleAds = false }) {
+  const sequence = adElement.getAttribute('sequence') || null;
+  if (sequence && !allowMultipleAds) {
+    //return;
+  }
```

Comment:
from what I understand, according to the specs ` only the first stand-alone Ad (with no sequence values) in the
requested VAST response is allowed.`. Since we don't know if there are multiple stand-alone ads here, we will also have to do the check once we have all the ads parsed.

Reply:
`//return;`
why the return is commented ? If we don't need it or if it's for later we should also comment the `if` and put a comment on top of it